utils/docker/Dockerfile.centos.8

```diff
@@ -85,6 +133,7 @@ RUN dnf -y install epel-release dnf-plugins-core && \
         python3-junit_xml \
         python3-numpy \
         python3-paramiko \
+        python3-pip \
```

Comment:
You don't need this since other changes in the last couple of weeks.

Reply:
Removing in next push.